tools/file_packager.py

```diff
@@ -1113,6 +1113,20 @@ def generate_js(data_target, data_files, metadata):
   function runMetaWithFS() {
     Module['addRunDependency']('%(metadata_file)s');
     var REMOTE_METADATA_NAME = Module['locateFile'] ? Module['locateFile']('%(metadata_file)s', '') : '%(metadata_file)s';
+    if (
+      typeof process === "object" &&
+      typeof process.versions === "object" &&
+      typeof process.versions.node === "string"
+    ) {
```

Comment:
Can we share the node detection code that already exists for fetch in the package data?

Reply:
How about we create a new `isNode` variable like the existing `isPthread` and `isWasmWorker` above?

Then we can also also great a local `var fs = ..` so we only need to require it once?